packages/react/src/primitives/Loader/Loader.tsx

```diff
@@ -58,10 +58,17 @@ const LoaderPrimitive: Primitive<LoaderProps, 'svg'> = (
     </g>
   );
 
+  size = size || 'default';
+  variation = variation || 'default';
+
   return (
     <View
       as="svg"
-      className={classNames(ComponentClassNames.Loader, className)}
+      className={classNames(
+        ComponentClassNames.Loader,
+        `${ComponentClassNames.Loader}--size-${size}--variation-${variation}`,
```

Comment:
For multiple modifiers, BEM syntax is to usually split each modifier into it's own class: https://css-tricks.com/bem-101/

Reply:
for modifiers where both are needed do you just create a new name for the modifier? for example the sliderfield has a variation and a size modifier that depends on each other.  Vertical sliders change the height with a size modifier and horizontal sliders change the width with a size modifier.

Reply:
Generally, modifiers should be mutually exclusive (size should not modify background color for instance). In the vertical/horizontal cases we can add a local CSS variable like `--slider-size: 1rem` which the size modifier modifies, and the orientation modifier uses in either `width` or `height`.